addons/common/functions/fnc_removeExclusionZone.sqf

```diff
@@ -0,0 +1,15 @@
+#include "..\script_component.hpp"
+
+params ["_area"];
+
+ISNILS(GVAR(EXCLUSION_ZONES), []);
+
+private _zoneIdx = GVAR(EXCLUSION_ZONES) findIf {
+	_area isEqualTo (if(_x isEqualType []) then [{_x#0}, {_x}]);
```

Comment:
ha, comparing with`_x#0` ... nice way to simplify the call.  👍 

just a thought: 

how about *not* requiring an exact match? 
we could look for an exclusion zone that covers the position being passed.

so like

```suggestion
	if(_x isEqualType []) then [{_area inArea _x}, {_area isEqualTo _x}]
```

Reply:
The idea of being less restrictive crossed my mind but thought it best to look for an exact match as, I'd assume, these APIs are being called programmatically where a position can be retrieved via marker or some fixed position.

Although, in a case where position is retrieved via an object that can move, it's a valid point. I'll implement and test at some point tonight or possibly early tomorrow afternoon.

Reply:
> [...] thought it best to look for an exact match as, I'd assume, these APIs are being called programmatically where a position can be retrieved via marker or some fixed position.

makes sense and actually... probably the safer approach: users might set overlapping exclusion zones, and in that case the "remove exclusion zone via `inArea`" option wouldnt be specific.

Reply:
I'll resolve this for now without making changes based on our discussion.

If this becomes a problem in the future we can certainly implement an `inArea` check as it would be backwards compatible. As you say, in a scenario where users add multiple overlapping exclusion zones it would be difficult to pick a candidate